qiskit/synthesis/discrete_basis/solovay_kitaev.py

```diff
@@ -155,17 +159,20 @@ def _recurse(self, sequence: GateSequence, n: int, check_input: bool = True) ->
             raise ValueError("Shape of U must be (3, 3) but is", sequence.shape)
 
         if n == 0:
-            return self.find_basic_approximation(sequence)
+            res = self.find_basic_approximation(sequence)
```

Comment:
IMO the `return` statement was a bit cleaner, but I don't have a very strong opinion 🙂

Reply:
Technically I did not need to change that, but for some reason I personally prefer this a bit more (but also I do not have a strong opinion).